### ISSUE TITLE
Lozad not working if images have alt="" in Chrome 76

### ISSUE THREAD

**Issue by ZCweb** [ApoorvSaxena/lozad.js#183]
For some accessibility situations I am having empty alt text with images. I noticed in Chrome - using version 76 - that the images will not load if the alt is null. As soon as I add alt text it works fine.

This problem is not in Firefox or Safari.

**Comment by aleehedl** [comment]
I have this problem as well.

**Comment by ZCweb** [comment]
Is anybody not having this problem? on the most popular browser on the internet? If you hover over the image it finally shows. Can someone please help I imagine this is a popular problem. I am willing to pay whomever whatever but this forum seems like the best place to resolve the issue.

**Comment by ApoorvSaxena** [comment]
@ZCweb @aleehedl could you please share a Codepen implementation, am unable to replicate the issue as displayed in the screenshot below while using Chrome v77

![alt-null-tag](https://user-images.githubusercontent.com/307583/65620882-48e2c400-dfe0-11e9-8c26-8673a6540112.png)

**Comment by ApoorvSaxena** [comment]
Unable to re-create, thus closing this issue

**Comment by mogularpb** [comment]
This issues appears when using a <picture> element. If the `data-alt` attribute equals an empty string, no alt attribute is added to the <img> element. This means that the image will still be attempted to be read by a screen reader, even though there is nothing meaningful to read - it will default to reading out the image file name. To avoid this an empty alt attribute should be added.
